icontribute-founder/icontribute-webapp
# WF-10.James

**What does this PR do?**
- Creates the landing page after a Volunteer Opportunity has been created
- Changed the NavBar at the top of the screen to match the latest Figma Design

**How to test this PR**
- Create a Volunteer Opportunity as an organization
- Verify the "http://localhost:3000/newOpportunityCreated" route

**Needs Improvement**
- Get Volunteer Opportunity Data from the previous page (using Redux?)
- Not all of the buttons work in the new Navigation Bar

![Screen Shot 2021-08-03 at 12 16 58 PM](https://user-images.githubusercontent.com/32142435/128050420-a2c970f2-bf30-4fa1-bdb4-95667f4dc3a6.png)
